### PR TITLE
[RobotCommander] Fill in in-code document where missing.

### PR DESCRIPTION
**Pull request by 130s**
- Added documents to methods where docs were absent.
- Loosely enforcing PEP8

Since I use `ipython` interpreter where methods' document can be referred to easily, I wanted to have more methods to show docs.

### CODE REVIEW

File: src/moveit_commander/robot.py [hunk 1]
```diff
@@ -177,12 +263,15 @@ def get_default_owner_group(self, joint_name):
...
+        to their properties.
+        """
         if name in self.get_group_names():
             return self.get_group(name)
         elif name in self.get_joint_names():
-            return self.Joint(self,name)
+            return self.Joint(self, name)
         elif name in self.get_link_names():
-            return self.Link(self,name)
+            return self.Link(self, name)
```
Comment (davetcoleman) [inline src/moveit_commander/robot.py:275]: is removing the space between variables an improvement for python style? i'm not a python expert. i see on line 265 (of the modified file) there is a space that was not removed:

```
def __getattr__(self, name):
```

seems inconsitent

**Comment by davetcoleman** [comment]
documentation! this is great

File: src/moveit_commander/robot.py (hunk 1 above)
Comment (130s) [inline src/moveit_commander/robot.py:275]: If I'm not mistaken, I'm padding space, not removing.

Comment (davetcoleman) [inline src/moveit_commander/robot.py:275]: ha, oops, you are

**Pull request merged**